anno_app.ipynb

```diff
@@ -300,9 +302,12 @@
     "\n",
     "\n",
     "class BinaryClassTask(ClassificationMixin):\n",
-    "    def __init__(self, labels, images, seed=None):\n",
+    "    def __init__(self, labels, images, unknown_option, seed=None):\n",
+    "        answer_choices = ['Yes', 'No']\n",
+    "        if unknown_option is not None:\n",
+    "            q_options.append(unknown_option)\n",
```

Comment:
should this `answer_choices` instead of `q_options`?

Reply:
yea good catch!